src/instruments/src/EFB/SimbriefApi/simbriefParser.ts

```diff
@@ -16,71 +16,72 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { ISimbriefData } from "./simbriefInterface";
+import { ISimbriefData } from './simbriefInterface';
 
-const simbriefApiUrl: URL = new URL("https://www.simbrief.com/api/xml.fetcher.php");
+const simbriefApiUrl: URL = new URL('https://www.simbrief.com/api/xml.fetcher.php');
 const simbriefApiParams = simbriefApiUrl.searchParams;
 
+// eslint-disable-next-line no-undef
 const getRequestData: RequestInit = {
```

Comment:
Why is this triggered? Because of RequestInit?

Reply:
adding `browser` to `env` should resolve this, but doesn't. this should get merged first